modules/heat_conduction/include/bcs/ADConvectiveHeatFluxBC.h

```diff
@@ -31,7 +31,7 @@ class ADConvectiveHeatFluxBC : public ADIntegratedBC<compute_stage>
 protected:
   virtual ADReal computeQpResidual() override;
 
-  /// Far-field temperatue fields for each component
+  /// Far-field temperatue variable
```

Comment:
temperatue -> temperatu**R**e :-) You missed that one :-)

Reply:
ug....